build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
   id 'pmd'
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.1.0'
-  id 'org.springframework.boot' version '2.7.10'
+  id 'org.springframework.boot' version '2.7.11'
```

Comment:
We will soon need update to java 17. Maybe it will be worth doing it sooner than later?